cmd/node/app_test.go

```diff
@@ -463,11 +463,17 @@ func (suite *AppTestSuite) validateLastATXActiveSetSize(app *SpacemeshApp) {
 }
 
 func TestAppTestSuite(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
 	//defer leaktest.Check(t)()
 	suite.Run(t, new(AppTestSuite))
 }
 
 func TestShutdown(t *testing.T) {
+	if testing.Short() {
```

Comment:
dont skip this

Reply:
same - this is only for short tests, which is not what happens when running `make test`, it just makes doing a quick run (3.5 minutes) of the short tests possible.

Reply:
I have to admit that I didn't mean to commit this, but since I already have I think it's a good thing.